packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -1,17 +1,224 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { render, click } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 
+// we're using this for multiple tests so we'll declare context once and use it when we need it.
+const setData = (context) => {
+  context.set('model', [
+    {
+      id: '1',
+      type: 'folk',
+      artist: 'Nick Drake',
+      album: 'Pink Moon',
+      year: '1972',
+    },
+    {
+      id: '2',
+      type: 'folk',
+      artist: 'The Beatles',
+      album: 'Abbey Road',
+      year: '1969',
+    },
+    {
+      id: '3',
+      type: 'folk',
+      artist: 'Melanie',
+      album: 'Candles in the Rain',
+      year: '1971',
+    },
+  ]);
+  context.set('columns', [
+    { key: 'artist', label: 'Artist' },
+    { key: 'album', label: 'Album' },
+    { key: 'year', label: 'Year' },
+  ]);
+  context.set('sortBy', 'artist');
+  context.set('sortOrder', 'asc');
+};
+const renderSortableTable = async () => {
```

Comment:
since the model is assigned directly, and not fetched, you don't need an async function. for this reason you can simplify the code even further, changing the `renderSortableTable` async function to a `const` string:
```
const hbsSortableTable = hbs`
  <Hds::Table ...>
    ...
  </Hds::Table>
`;
```
and then call the render function passing directly the string itself
```
await render(hbsSortableTable);
```